specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -24,7 +24,7 @@ options:
   "@azure-tools/typespec-java":
     package-dir: "azure-resourcemanager-contoso"
     namespace: "com.azure.resourcemanager.contoso"
-    service-name: "contoso"
+    service-name: "Contoso"
```

Comment:
Maybe adding a comment will be a good reminder for the users when they update its value.

Reply:
You might also want to update [type-spec/azure ARM template](https://github.com/Azure/typespec-azure/blob/457a8bcd428c6fac3e6c9994cb5437b4c4fc2575/eng/feeds/arm/tspconfig.yaml#L19), which uses a lowercase service name.

Reply:
About comment, can I do e.g.
```
    service-name: "Contoso" // human-readable service name, whitespace allowed
```
(thought about it but didn't see anyone do this)

Emm, I am not sure what's the syntax in the feed file. Would this work?
```
    service-name: "{{#toUpperCase}}{{parameters.ServiceNamespace}}{{/toUpperCase}}"
```

Reply:
Updated the typespec-azure https://github.com/Azure/typespec-azure/pull/2600